mirrord/config/src/lib.rs

```diff
@@ -327,15 +331,50 @@ pub struct LayerConfig {
 }
 
 impl LayerConfig {
+    /// Given an encoded complete config from the [`MIRRORD_RESOLVED_CONFIG_ENV`]
+    /// env var, attempt to decode it into [`LayerConfig`].
+    /// Intended to avoid re-resolving the config in every process mirrord in loaded into
+    pub fn from_env_var(encoded_value: String) -> Result<Self, ConfigError> {
+        let decoded = BASE64_STANDARD
+            .decode(encoded_value)
+            .map_err(|error| ConfigError::EnvVarDecodeError(error.to_string()))?;
+        let serialized = std::str::from_utf8(&decoded)
+            .map_err(|error| ConfigError::EnvVarDecodeError(error.to_string()))?;
+        Ok(serde_json::from_str::<Self>(serialized)?)
+    }
+
+    pub fn to_env_var(&self) -> Result<String, ConfigError> {
+        let serialized = serde_json::to_string(self)
+            .map_err(|error| ConfigError::EnvVarEncodeError(error.to_string()))?;
+        Ok(BASE64_STANDARD.encode(serialized))
+    }
+
+    pub fn update_env(&self) -> Result<(), ConfigError> {
```

Comment:
```suggestion
    pub fn update_env_var(&self) -> Result<(), ConfigError> {
```

Nit, since the others are `*_env_var`.